Index: sonic-pi-3.2.2/app/server/ruby/bin/sonic-pi-server.rb
===================================================================
--- sonic-pi-3.2.2.orig/app/server/ruby/bin/sonic-pi-server.rb
+++ sonic-pi-3.2.2/app/server/ruby/bin/sonic-pi-server.rb
@@ -35,6 +35,14 @@ require 'memoist'

 include SonicPi::Util

+# Start compton if using Raspberry Pi: enable transparency
+if os==:raspberry
+  STDOUT.puts  "Starting compton for Raspberry Pi transparency"
+  cmd="/usr/bin/compton"
+  compton_pid = spawn("exec #{cmd}",[:out,:err]=>"/dev/null")
+  register_process(compton_pid)
+end
+
 ## This is where the server starts....
 STDOUT.puts "Sonic Pi server booting..."
 STDOUT.puts "This is version 3.2.0 running on Ruby #{RUBY_VERSION}."
